src/lib/extract/extractFilter.ts

```diff
@@ -46,7 +46,9 @@ export const extractFeColorMatrix = (
 
   if (props.values !== undefined) {
     if (Array.isArray(props.values)) {
-      extracted.values = props.values;
+      extracted.values = props.values.map((num) =>
```

Comment:
Why do we do those changes in this PR?

Reply:
Reverted in [31af51b](https://github.com/software-mansion/react-native-svg/pull/2392/commits/31af51bf062a2054b5082d2693829a2f7b3e46a4)